eng/Subsets.props

```diff
@@ -483,6 +483,13 @@
     <ProjectToBuild Include="$(LibrariesProjectRoot)pretest.proj" Category="libs"  />
   </ItemGroup>
 
+  <!-- For running tests with TestReadyToRun, we need the R2R-precompiled runtime prior to building the tests. -->
+  <ItemGroup Condition="$(_subset.Contains('+libs.pretest+')) and '$(TestReadyToRun)' == 'true'">
+    <R2RPrecompiledRuntimeProject Include="$(InstallerProjectRoot)pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.sfxproj" />
+    <R2RPrecompiledRuntimeProject Include="$(InstallerProjectRoot)pkg/sfx/bundle/Microsoft.NETCore.App.Bundle.bundleproj" />
+    <ProjectToBuild Include="@(R2RPrecompiledRuntimeProject)" Category="packs" />
+  </ItemGroup>
```

Comment:
This dependency shouldn't be expressed here in the Subsets file. The Subsets file is intended to promote build entry points but not to define strong dependencies. Why isn't the change in pretest.proj sufficient? That runs before any of the tests are built.

Reply:
I will try looking into seeing if I can make it work in `pretest.proj` somehow. I regret to say that the change you suggested there earlier didn't work.

Reply:
Actually, never mind. I think I know what I did wrong there.